### DISCUSSION TITLE
Why does this look like a ripoff of Apollo?

### DISCUSSION THREAD

**Discussion by ORIOLESFan02** [aeharding/voyager#1268] [Q&A]
I understand this isnt a normal issue, but this app seems like a compelete ripoff of Christian Sellig's Apollo. Why is that?
Edit: was permission given to use Apollo's design?

**Comment by aeharding** [comment]
Hi!

I love Apollo, and I was sad that it was going to be shut down. That's why I built Voyager. :)

There's maybe a bit more context in this thread: https://lemmy.world/post/8784178?scrollToComments=true
